analyzer/src/funTest/kotlin/PackageManagerFunTest.kt

```diff
@@ -73,7 +73,8 @@ class PackageManagerFunTest : WordSpec({
         "spdx-project/project.spdx.yml",
```

Comment:
Commit message:
Do not use `Implements: <Link to Issue>`.

Use an autoclosing keyword like: `Resolves #10182.`